pkg/virt-controller/services/template.go

```diff
@@ -453,6 +457,10 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, t
 
 	serviceAccountName := ""
 
+	if val, ok := vmi.GetAnnotations()[ISTIO_INJECT_ANNOTATION]; ok && strings.ToLower(val) == "true" {
+		serviceAccountName = DEFAULT_SERVICE_ACCOUNT_NAME
```

Comment:
Is it sufficient for Istio to inject any service account?
We can maybe do something like this:
```

automount := true
pod.Spec.AutomountServiceAccountToken = &automount
```
if for any reason the default account is different.

Reply:
>Is it sufficient for Istio to inject any service account?

To my understanding, yes. [there is a requirement for NET_RAW and NET_ADMIN](https://istio.io/latest/docs/ops/deployment/requirements/), but we intent to only use Istio with CNI Plugin. 
They don't mention any other requirement for the service account.

So I believe you have a point. [Updated the accordingly to your suggestion](https://github.com/kubevirt/kubevirt/compare/eb0cee1722d0ca2845285dc7c66548fdaa75264d..262e988b3b60906d77210e48974869b290dc2e22)

Reply:
Right, thanks. [Done](https://github.com/kubevirt/kubevirt/compare/262e988b3b60906d77210e48974869b290dc2e22..73ac0b974aec61580368030c887b39c44f350241), also removed the now unused constant.